bundles/com.espressif.idf.sdk.config.ui/src/com/espressif/idf/sdk/config/ui/SDKConfigurationEditor.java

```diff
@@ -611,7 +613,10 @@ protected void renderMenuItems(KConfigMenuItem selectedElement)
 					: false);
 			Object newConfigValue = modifiedJsonMap.get(configKey);
 			String helpInfo = kConfigMenuItem.getHelp();
-
+			if (type.equals(IJsonServerConfig.MENU_TYPE))
+			{
+				isVisible = true;
+			}
```

Comment:
Can we move this validation here? https://github.com/espressif/idf-eclipse-plugin/blob/master/bundles/com.espressif.idf.sdk.config.core/src/com/espressif/idf/sdk/config/core/KConfigMenuItem.java#L107-L118

Reply:
done